### PR TITLE
Spelling: Compound noun error, A username

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: src/Views/AccountView.vala [hunk 1]
```diff
@@ -205,9 +205,9 @@ public class Installer.AccountView : AbstractInstallerView {
             return true;
         } else {
             if (username_is_taken) {
-                username_error_revealer.label = _("Username is already taken");
+                username_error_revealer.label = _("Chosen username already taken");
```
Comment (jeremypw) [inline src/Views/AccountView.vala]: Should this not be "The chosen usename has already been taken" if you want 'proper' grammar?

File: src/Views/AccountView.vala [hunk 2]
```diff
@@ -205,9 +205,9 @@ public class Installer.AccountView : AbstractInstallerView {
...
             } else if (!username_is_valid) {
-                username_error_revealer.label = _("Username can only contain lowercase letters and numbers, without spaces");
+                username_error_revealer.label = _("A username can only contain lowercase letters and numbers, without spaces");
```
Comment (jeremypw) [inline src/Views/AccountView.vala]: This should be "A username must only contain ...".

File: src/Views/AccountView.vala (hunk 1 above)
Comment (comradekingu) [reply src/Views/AccountView.vala]: It can be either, but it can't be the former.

File: src/Views/AccountView.vala (hunk 2 above)
Comment (comradekingu) [reply src/Views/AccountView.vala]: You would have to say "A username must contain no other characters than letters and numbers, without the use of spaces in between" for that to work.

Comment (jeremypw) [reply src/Views/AccountView.vala]: To me (and I am English) the meaning is the same.

File: src/Views/AccountView.vala (hunk 1 above)
Comment (jeremypw) [reply src/Views/AccountView.vala]: I agree, it is a balance between being succinct and being unambiguously translatable I guess.

**Pull request merged**